voltdb/src/main/java/org/voltdb/ycsb/package-info.java

```diff
@@ -0,0 +1,7 @@
+/**
+ * VoltDB integration with YCSB.
+ * 
+ * @author srmadscience / VoltDB
+ *
+ */
+package org.voltdb.ycsb;
```

Comment:
this should be in `com.yahoo.ycsb.db.voltdb` or a package under that

Reply:
Should be fixed in 0a534e8434d7e854901fbcf825df0403cc985ba3 .